### PR TITLE
Updated c++-quiz-de.md file - HACKTOBERFEST

### PR DESCRIPTION
**Pull request by surbhi2408**
Added one question for C++.

### CODE REVIEW

File: c++/c++-quiz-de.md [hunk 1]
```diff
@@ -195,3 +195,12 @@ void *ptr;
...
 - [ ] indem er einfach den C-Code aufruft
+
+#### Q16. What is a class template?
+
+- [x] It’s a class written with the generic programming, specifying behavior in terms of type parameter rather than specific type.
+- [ ] It’s a blank superclass intended for inheritance and polymorphism.
+- [ ] It’s class that only consists of member variable, with no constructor, destructor nor member functions.
+- [ ] It’s skelton source code for a class where the programmer has to fill in specific parts to define the data types and algorithms used.
+
+  [References](https://www.ibm.com/docs/en/zos/2.2.0?topic=only-class-templates-c)
```
Comment (auchynnikau) [inline c++/c++-quiz-de.md:206]:
```suggestion
[References](https://www.ibm.com/docs/en/zos/2.2.0?topic=only-class-templates-c)
```

Comment (surbhi2408) [reply c++/c++-quiz-de.md]: I have resolved the changes that were told. Now is it fine?

**Review by Ebazhanov — APPROVED** [review]
Looks good, thank you @surbhi2408 for your contribution! 👍

**Comment by Ebazhanov** [comment]
@all-contributors please add @surbhi2408 for design, code

**Pull request merged**